msgpack-jackson/src/main/java/org/msgpack/jackson/dataformat/MessagePackGenerator.java

```diff
@@ -513,6 +513,13 @@ private void writeByteArrayTextKey(byte[] text, int offset, int len) throws IOEx
         addValueNode(new String(text, offset, len, DEFAULT_CHARSET));
     }
 
+    // TODO: Uncomment
+    //@Override
+    //public void writeFieldId(long id) throws IOException
+    //{
+    //    addKeyToStackTop(id);
```

Comment:
☝️ Correction. Comment was drafted for older msgpack.
```suggestion
    //    addKeyNode(id);
```